Documentation/Browser/AddNewIDLFile.md

```diff
@@ -0,0 +1,31 @@
+# Adding a new IDL file
+
+Serenity's build system does a lot of work of turning the IDL from a Web spec into code, but there are a few things you'll need to do yourself.
+
+For the sake of example, let's say you're wanting to add the `HTMLDetailsElement`.
+
+1. Create `LibWeb/HTML/HTMLDetailsElement.idl` with the contents of the IDL section of the spec. In this case, that would be:
+```
+[Exposed=Window]
+interface HTMLDetailsElement : HTMLElement {
+  [HTMLConstructor] constructor();
+
+  [CEReactions] attribute boolean open;
```

Comment:
we use 4 spaces for IDL files, and even though it's not enforced through linting, let's encourage it here:

```suggestion
    [HTMLConstructor] constructor();

    [CEReactions] attribute boolean open;
```